kubernetes/deployment-staging.tmpl

```diff
@@ -330,6 +346,7 @@ spec:
       containers:
         - name: panoptes-staging-redis
           image: redis:6.0.5
+          args: ['--save ""', '--appendonly no']
```

Comment:
Did you mean to include this here? Don't we want redis data to be persistent for Panoptes?